src/plugins/github/parseReferences.js

```diff
@@ -49,12 +51,14 @@ function findNumericReferences(textBlock: string): ParsedReference[] {
 }
 
 function findUsernameReferences(textBlock: string): ParsedReference[] {
+  const pairedWithPattern = `(?:\\W|^)(?:P|p)aired(?:-| )(?:w|W)ith:? (@(?:${githubOwnerPattern}))(?=\\W|$)`;
```

Comment:
Moderate preference for respecting the max line length, but I won't insist on it.